modules/developer_manual/examples/core/scripts/ruby/decline-pending-share.rb

```diff
@@ -0,0 +1,14 @@
+require 'net/http'
+require 'uri'
+
+base_uri = '{oc-examples-server-url}/ocs/v1.php/apps/files_sharing/api/v1/'
+uri = URI("#{base_uri}/shares/pending/115468")
```

Comment:
kind of weird that the rest of the example has a "/pending/1" except this one.

Reply:
Thanks for spotting this.